package.json

```diff
@@ -19,14 +19,20 @@
     "ckeditor5",
     "ckeditor 5"
   ],
+  "dependencies": {
+    "lodash-es": "^4.17.21"
+  },
+  "peerDependencies": {
+    "vue": "^3.0.0",
+    "ckeditor5": ">=42.0.0 || ^0.0.0-nightly"
```

Comment:
If none is installed, what will npm install by default?

Reply:
I tested it with `>=41.0.0 || ^0.0.0-nightly` (because `v42` is not yet released) and it installed `v41` by default.

I also tested it with `yarn` and `pnpm`, but not surprises.